pkg/virtualKubelet/reflection/storage/storage_suite_test.go

```diff
@@ -0,0 +1,198 @@
+// Copyright 2019-2021 The Liqo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package storage
```

Comment:
Nit. I personally prefer using a separate test package, since it avoids to pollute the original one (e.g., with the variables defined below). Yet, that is mostly a matter of taste (feel free to ignore).